src/utils.jl

```diff
@@ -0,0 +1,4 @@
+
+_round(v::AbstractArray) = _round.(v)
+_round(x::Integer) = x
+_round(x) = round(x; digits=4)
```

Comment:
Maybe we could use "significant digits" since the scale might be off by a lot?  also 4 digits seems a little aggressive.

Reply:
Significant digits isn't great for large numbers .... but dropping to 2 is okay (also for space purposes)